src/Classes/Controllers/GetEventsInfoController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+
+namespace Portal\Controllers;
+
+use Slim\Http\Request;
+use Slim\Http\Response;
+
+class GetEventsInfoController
```

Comment:
Suggest renaming to `GetEventController` to remove redundant word and consistency with codebase.

Reply:
Good point will rename the factory too.